openapi/platform/platform.yaml

```diff
@@ -6776,6 +6776,65 @@ components:
                 x-stoplight:
                   id: yflmky9gif6ro
                 type: string
+            adminNotes:
+              type: string
+              x-stoplight:
+                id: tnzahuoq5fsyi
+              description: Administration notes for the account.
+            socialUrls:
+              type: object
+              x-stoplight:
+                id: 8bta15bsc842h
+              description: The Social URLs for a Sales Account.
+              properties:
+                googleplusUrl:
+                  type: string
+                  x-stoplight:
+                    id: ojx06t4cb7thm
+                  format: uri
+                  description: The URL for a Sales Account's Google Plus page.
+                linkedinUrl:
+                  type: string
+                  x-stoplight:
+                    id: co78ugcn9hlc0
+                  format: uri
+                  description: The URL for a Sales Account's LinkedIn page.
+                foursquareUrl:
+                  type: string
+                  x-stoplight:
+                    id: tsw7lpw0p4neo
+                  format: uri
+                  description: The URL for a Sales Account's Foursquare page.
+                xUrl:
```

Comment:
`xUrl` is going to be a confusing name. It is common practice to prefix headers and field names with `x-` to indicate that they are custom. Scanning the docs it took me a moment to realize this was twitter. 

x.com currently redirects to twitter.com. I wonder if we should keep the old name of `twitterURL`

Reply:
chatted with our design team, and they suggest we stay with "X" to be consistent as everything is being converted to use "X" instead of twitter

Reply:
`xFormerlyTwitterUrl` pls